packages/amplify-graphql-default-value-transformer/src/__tests__/amplify-grapphql-default-value-transformer.test.ts

```diff
@@ -338,4 +198,90 @@ describe('DefaultValueModelTransformer:', () => {
     expect(out.resolvers['Mutation.createNote.init.1.req.vtl']).toBeDefined();
     expect(out.resolvers['Mutation.createNote.init.2.req.vtl']).toBeUndefined();
   });
+
+  it.each([{ strategy: mockSqlDataSourceStrategy() }])(
+    'throws if auto-increment is implied on a non-Postgres datasource',
+    ({ strategy }) => {
+      const schema = `
+      type CoffeeQueue @model {
+        id: ID! @primaryKey
+        orderNumber: Int! @default
+        name: String
+      }`;
+      expect(() => {
+        testTransform({
+          schema: schema,
+          transformers: [new ModelTransformer(), new DefaultValueTransformer(), new PrimaryKeyTransformer()],
+          dataSourceStrategies: constructDataSourceStrategies(schema, strategy),
+        });
+      }).toThrow('The @default directive requires a value property on non Postgres datasources.');
```

Comment:
1. Can you confirm that a `@default(value: ...)` works as expected on a mysql data source? (That is, it passes specified value in the input if the mutation input doesn't include it; and it allows overrides of the default if it IS specified). Best way to confirm that would be a test (possibly an already existing one!)
2. Do we have a test (possibly an already existing one!) to assert that `@default(value:...)` works for postgres?

Reply:
The only e2e usage of `@default` I could find only used DDB. I modified the general sql models test to utilize `@default(value)` which should verify behavior on both sources.